packages/create-plugin/templates/provisioning-template/README.md

```diff
@@ -0,0 +1,20 @@
+# Plugin name
```

Comment:
I think we can auto-generate this and the description based on other template files